latte/latte.Config.json

```diff
@@ -14,7 +14,7 @@
 
     "LatteAzure": 
     {
-        "Iterations"     : 15,
+        "Iterations"     : 20,

```

Comment:
Please ping me offline to discuss.   15 iteration is already a very large amount.  It's not clear to me how adding more helps to normalize, vs modifying the run to execute for longer time as long haul.  Which is a more correct validation methodology.